lib/ansible/plugins/test/directory.yml

```diff
@@ -0,0 +1,21 @@
+DOCUMENTATION:
+  name: directory
+  author: Ansible Core
+  version_added: "2.5"
+  short_description: does the path resolve to an existing directory
+  description:
+    - Check if the provided path maps to an existing directory on the controller's filesystem (localhost)
+  options:
+    _input:
+      description: a path
+      type: path
+
+EXAMPLES:
+  vars:
+    my_etc_hosts_not_a_dir: "{{ '/etc/hosts' is directory}}"
+    list_of_files: "{{ list_of_paths | reject('directory') }}"
+
+RETURN:
+  _value:
+    description: Returns C(True) the path corresponds to an existing directory on the filesystem on the controller, c(False) if otherwise
```

Comment:
```suggestion
    description: Returns C(true) the path corresponds to an existing directory on the filesystem on the controller, C(false) if otherwise.
```

Reply:
```suggestion
    description: Returns C(True) the path corresponds to an existing directory on the filesystem on the controller, c(False) if otherwise.
```